docs/api-reference/map.md

```diff
@@ -54,6 +55,8 @@ Returns the native [Map](https://docs.mapbox.com/mapbox-gl-js/api/map/) instance
 
 ## Properties
 
+Aside from the props listed below, the `Map` component supports all parameters of the [Map](https://docs.mapbox.com/mapbox-gl-js/api/map/#map-parameters) constructor. Beware that this is not an exhaustive list of all props. Different base map libraries may offer different options and default values. When in doubt, refer to your base map library's documentation.

```

Comment:
I think we should link to both mapbox and maplibre parameters.

Reply:
I will make another pass on all the documentation.